### PR TITLE
adding lstmcpipe package

### PR DESCRIPTION
**Pull request by vuillaut**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vuillaut** [comment]
Maintainer here commenting 👋

**Comment by vuillaut** [comment]
@conda-forge/python, ready for review!
-  `python >=3.9` is required by lstchain - I had to add the requirement
- The windows build is failing because of lstchain - should I skip the build?

**Comment by vuillaut** [comment]
@conda-forge/python, ready for review!

File: recipes/lstmcpipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.9
+    - lstchain >=0.10.0
```
Comment (ocefpaf) [inline recipes/lstmcpipe/meta.yaml:37]: Looks like the metadata is:

```suggestion
    - lstchain >=0.9.0
```

any reason to be more restrivtive?

Comment (vuillaut) [reply recipes/lstmcpipe/meta.yaml:37]: lstchain has no conda package before 0.10.0

Comment (ocefpaf) [reply recipes/lstmcpipe/meta.yaml:37]: That should not be a problem b/c 0.10.0 will be found and, if folks ever want to use this recipe outside of conda-forge with 0.9.0, they don't need to dig into the metadata to fix it.

Comment (vuillaut) [reply recipes/lstmcpipe/meta.yaml]: I see, thanks, I fixed it

**Pull request merged**